test/mobile/custom_build/predictor.cpp

```diff
@@ -0,0 +1,52 @@
+#include <iostream>
+#include <string>
+#include <torch/script.h>
+
+using namespace std;
+
+namespace {
+
+struct MobileCallGuard {
+  // AutoGrad is disabled for mobile by default.
+  torch::autograd::AutoGradMode no_autograd_guard{false};
+  // Disable graph optimizer to ensure list of unused ops are not changed for
+  // custom mobile build.
+  torch::jit::GraphOptimizerEnabledGuard no_optimizer_guard{false};
```

Comment:
The reasoning here makes sense, but in general, this seems a bit questionable, since I imagine that one might quite reasonably want to optimize graphs for mobile build. Wouldn't it be "more correct" to optimize first, and then compute the list of unused ops? Can this be conveniently supported?

I understand the goal of this PR is to get an e2e CI running, so this wouldn't be blocking, but it's something to think about...